data-library/library.sh

```diff
@@ -20,42 +20,19 @@ function setup {
     apt update
     apt-get install -y python3-pip python3-distutils gdal-bin libgdal-dev curl git unzip
     apt-get -y install --reinstall build-essential
-
-    python3 -m pip install poetry
-    poetry config virtualenvs.create false --local
-
-    case $1 in 
-        dev) ;;
-        *) option="--no-dev";;
-    esac
-    gdal_version=$(gdal-config --version)
-    gdal_poetry_version=$(poetry_version gdal)
-
-    if [ $gdal_version != $gdal_poetry_version ]; then
-        echo "mismatch between installed gdal binary and gdal version in pyproject.yaml"
-        echo "update poetry gdal version to $gdal_version"
-        exit 1
-    fi
-    poetry install $option
+    
+    pip install -e . -c ${1:-constraints.txt}
```

Comment:
still editable install so that local file system (and template files) can be referenced - code currently uses path logic relative to current file, which long term should probably be removed.

If we got rid of logic around local file structure, would need something along the lines of having a default path of `library/templates` (or we could move the templates up a folder and just default to a `templates` folder in the running directory), and still have ability to provide full path to template rather than name of template, which is currently supported